yarn-project/key-store/src/test_key_store.ts

```diff
@@ -58,16 +60,23 @@ export class TestKeyStore implements KeyStore {
     await this.#keys.set(`${accountAddress.toString()}-public_keys_hash`, publicKeysHash.toBuffer());
 
     // Naming of keys is as follows ${from}-${to}_m
-    await this.#keys.set(`${accountAddress.toString()}-nsk_m`, masterNullifierSecretKey.toBuffer());
     await this.#keys.set(`${accountAddress.toString()}-ivsk_m`, masterIncomingViewingSecretKey.toBuffer());
     await this.#keys.set(`${accountAddress.toString()}-ovsk_m`, masterOutgoingViewingSecretKey.toBuffer());
     await this.#keys.set(`${accountAddress.toString()}-tsk_m`, masterTaggingSecretKey.toBuffer());
+    // The key of the following is different from the others because the buffer can store multiple keys
+    await this.#keys.set(`${accountAddress.toString()}-ns_keys_m`, masterNullifierSecretKey.toBuffer());
 
-    await this.#keys.set(`${accountAddress.toString()}-npk_m`, publicKeys.masterNullifierPublicKey.toBuffer());
+    await this.#keys.set(`${accountAddress.toString()}-np_keys_m`, publicKeys.masterNullifierPublicKey.toBuffer());
```

Comment:
Why is the naming format changed so much here? I get that it can be multiple, but it is weird that the format is not followed, could be `nsk_m_s` and `npk_m_s` with the s marking the plural. It is a nit, but 🤷

Reply:
You have a point. Just wanted to communicate that it can contain multiple keys here.